<?xml version="1.0"?>
<!-- Note: example launch file. Do not modify! -->
<launch>
  <arg name="loop_rate" default="100"/>
  <arg name="wait_for_transform" default="0.1"/>
  <arg name="use_remote_machine" default="false"/>
  <arg name="machine" default="None"/>

  <group if="$(arg use_remote_machine)">
    <node pkg="udom_geometric_transformation" type="pose_error_calculator"
          name="pose_error_calculator" output="screen" machine="$(arg machine)">
      <!-- Node cycle rate (in Hz). -->
      <param name="loop_rate" type="double" value="$(arg loop_rate)"/>

      <!-- Maximum duration to wait for a transform (in seconds). -->
      <param name="wait_for_transform" type="double" value="$(arg wait_for_transform)"/>
    </node>
  </group>

  <group unless="$(arg use_remote_machine)">
    <node pkg="udom_geometric_transformation" type="pose_error_calculator"
          name="pose_error_calculator" output="screen">
      <!-- Node cycle rate (in Hz). -->
      <param name="loop_rate" type="double" value="$(arg loop_rate)"/>

      <!-- Maximum duration to wait for a transform (in seconds). -->
      <param name="wait_for_transform" type="double" value="$(arg wait_for_transform)"/>
    </node>
  </group>
</launch>
